python/llm/example/CPU/Applications/autogen/README.md

```diff
@@ -1,6 +1,5 @@
 ## Running AutoGen Agent Chat with BigDL-LLM on Local Models
-In this example, we use BigDL adapted FastChat to run [AutoGen](https://microsoft.github.io/autogen/) agent chat with 
-local large language models.
+In this example, we use BigDL-adapted FastChat to run [AutoGen](https://microsoft.github.io/autogen/) teachable  agent chat with local large language models. This agent is capable of addressing the short-term memory limitation in conversational LLM chatting. It stores the user teachings to long-term memory saved on disk. Those memories will be retrieved into the chat only as needed. This allows the user to teach the teachable agent many facts, preferences, and skills and have it remember them in later chats. This example demonstrates how `Teachability` can be added to the agent from the user. For detailed illustration, please refer to the original documentation from AutoGen [Teachability Example](https://github.com/microsoft/autogen/blob/main/notebook/agentchat_teachability.ipynb).
```

Comment:
This example is adapted from the [Official AutoGen Teachablility Example](https://github.com/microsoft/autogen/blob/main/notebook/agentchat_teachability.ipynb). 

In this exmaple, ...